templates/data-provider/result-placeholder.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import styles from './styles.module.css'
+
+import Placeholder from 'modules/placeholder'
+import Search from 'modules/search-layout'
+
+const ResultPlaceholder = ({ ...passProps }) => (
+  <Placeholder tag={Search.Result} className={styles.result} {...passProps}>
+    <div className={styles.resultImage}>
+      <img
+        src="https://core.ac.uk/images/document-placeholder.svg"
+        alt="Loading placeholder"
+      />
+    </div>
+    <div className={styles.resultContent}>
+      <Placeholder.Heading />
+      <Placeholder.Line />
+      <Placeholder.Spacer />
+      <Placeholder.Line />
+      <Placeholder.Line />
+      <Placeholder.ShortLine />
+    </div>
```

Comment:
You may find useful oacore/design#349

Reply:
I will wait for it.

Reply:
The idea behind the PR is complete preserving the layout but changing only one flag. I cannot find any other way to test it than moving search result component to design modules and do it there. I can do this.